components/Cards/README.md

```diff
@@ -14,28 +14,79 @@ api_doc_root: true
   <img src="docs/assets/cards.png" alt="Cards" width="375">
 </div>
 
-Cards provides an card control and a card collection view cell that adhere to Material Design
-layout and styling.
+Cards are contained surfaces that serve as an entry point to more detailed information, with flexible visual parameters.
+
+They can be used as a standalone UIControl, or as a UICollectionViewCell in a UICollectionView.
+Cards are meant to be interactive, and shouldn't be used solely for displaying purposes.
+
+Cards adhere to Material Design layout and styling.
 
 ## Design & API Documentation
 
 <ul class="icon-list">
-  <li class="icon-list-item icon-list-item--spec"><a href="https://material.io/guidelines/components/cards.html">Material Design guidelines: Cards</a></li>
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/cards.htm">Material Design guidelines: Cards</a>
+  </li>
 </ul>
 
+## Installation
+
+### Requirements
```

Comment:
We are not putting requirements in individual components any more.  
They have been moved to the main README.md.
We can remove the ### Requirements section.